Examples/LambdaFunctions/Sources/CurrencyExchange/main.swift

```diff
@@ -166,8 +169,9 @@ struct ExchangeRatesCalculator {
         let ratesByCurrencyCode: [String: Decimal?] = Dictionary(uniqueKeysWithValues: try currencyCodes.map {
             let xpathCurrency = $0.replacingOccurrences(of: "'", with: "&apos;")
             if let rateString = try document.nodes(forXPath: "/exchangeRateMonthList/exchangeRate/currencyCode[text()='\(xpathCurrency)']/../rateNew/text()").first?.stringValue,
-                // We must parse the decimal data using the UK locale, not the system one.
-                let rate = Decimal(string: rateString, locale: self.locale) {
+               // We must parse the decimal data using the UK locale, not the system one.
+               let rate = Decimal(string: rateString, locale: self.locale)
+            {
```

Comment:
yeah these are pretty weird... why does it cut down the {

Reply:
yea this is odd, is there an option to turn this off?